modules/services/agentless-scanning/variables.tf

```diff
@@ -8,6 +8,11 @@ variable "trusted_identity" {
   description = "The name of sysdig trusted identity"
 }
 
+variable "agentless_account_id" {
```

Comment:
you would probably need a change on the onboarding side as well right?

Reply:
Yes, for Production use (i.e. Customer use), this is going to be a fixed string ala `source`.
All Customers will leverage the same infrastructure we created to leverage Snapshots.

For our internal use, we'll need to point all these resources to a development "scanner" account in order to test workers.